konduit-serving-models/konduit-serving-tvm/pom.xml

```diff
@@ -53,6 +53,16 @@
       <groupId>org.nd4j</groupId>
       <artifactId>nd4j-tvm</artifactId>
       <version>${nd4j.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.bytedeco</groupId>
+          <artifactId>dnnl-platform</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.bytedeco</groupId>
+          <artifactId>dnnl</artifactId>
+        </exclusion>
```

Comment:
Was this necessary? I prefer not to use exclusions unless absolutely necessary.

Reply:
it's necessary, otherwise the build will have error.

Reply:
@ndthanhit  can you add a comment or something showing the dependency trace?